resolwe_bio/tests/processes/test_enrichment.py

```diff
@@ -21,6 +23,7 @@ def test_go_enrichment_dicty(self):
         enrichment = self.run_process('goenrichment', inputs)
         self.assertJSON(enrichment, enrichment.output['terms'], '', 'go_enriched_terms_dicty.json.gz')
 
+    @with_resolwe_api_host
```

Comment:
I think even better would be to extend the `with_resolwe_api_host` decorator to also work with the whole Class.